src/Data/Sequence/Strict.hs

```diff
@@ -84,7 +84,7 @@ infixl 5 :|>
 -- value to WHNF.
 newtype StrictSeq a = StrictSeq { getSeq :: Seq a }
   deriving stock (Eq, Ord, Show)
-  deriving newtype (Foldable, Semigroup, Serialise)
+  deriving newtype (Foldable, Semigroup, Serialise, Traversable)
```

Comment:
This is not going to maintain strictness

Reply:
It's not even going to compile. But it's out of date...